src/connectedk8s/azext_connectedk8s/custom.py

```diff
@@ -307,9 +376,13 @@ def get_helm_registery(profile, location):
     try:
         response = requests.post(get_chart_location_url, params=query_parameters, headers=header_parameters)
```

Comment:
Should we log which release train is being used and so on?

Reply:
For customer's it will always be stable, so no need to log